tests/integration/cocoon/data/port_group_tests/test_1_pg_create_with_portfolios.csv

```diff
@@ -0,0 +1,5 @@
+,PortGroupCode,PortGroupDisplayName,FundCode,
+0,PG_TEST1,Test Portfolio Group,TEST_COM1,
+1,PG_TEST1,Test Portfolio Group,TEST_COM2,
+2,PG_TEST2,Test Portfolio Group,TEST_COM3,
+4,PG_TEST3,Test Portfolio Group,TEST_COM4,
```

Comment:
Some of these test files have a leading index column, and others are mssing. Is this necessary/deliberate?